### ISSUE TITLE
mysql_num_rows ... Ride-In-Peace

### ISSUE THREAD

**Issue by rainbat** [Desertsnowman/DB-Toolkit#18]
Hello Friends-of-DBToolkit,

couple of years ago I started a couple of projects on the basis of DB-Toolkit and it helped me well over the years. I spent much time in the array structures of David and also in the Source Code and I'm thinking about porting the whole thing to a very basic level. 

I will remain compatible with the Datamodel (the arrays) and simply replace the frontend with bootstrap and the backend in simple text configuration files.

@Desertsnowman: I'm comfortable with the datamodel, but will you point me to the right files when I redo the frontend ? Like for example where the code is to render the Pulldown Fields that reference lookup tables et. ?

Are there other People interested in this ?

Cheers and Greeting from Zuerich, Dominique

**Comment by Desertsnowman** [comment]
yes I'll help out where needed. (when time allows)
I don't remember much about it but should be easy enough to figure out.

**Comment by Desertsnowman** [comment]
@rainbat actually, how about we work together on making a new port?

**Comment by rainbat** [comment]
yes lets do that ! wanna connect on skype ? i am zuerich0910

**Comment by rainbat** [comment]
@aoloe: Ale you already helped in decoding the array into reports that are readable by the customer, would love to have you on board

**Comment by aoloe** [comment]
yeah, wordpress needs a good and free form builder!

**Comment by Desertsnowman** [comment]
@aoloe - https://github.com/Desertsnowman/Caldera-Forms

**Comment by Desertsnowman** [comment]
@rainbat - actually, maybe we can use Caldera Forms as the form part.

**Comment by aoloe** [comment]
oh, never heard of caldera form...

i skimmed through the website and i could not really tell if it can just mail / let further process form entries or if it lets manage the entries in wordpress.

personally i worked with gravity forms and formidable pro (and little bit with db-toolkit...) and i liked both of them, up to the fact that they are not really free.
and since i need them for site about free software this is a problem i'd like to see resolved...

**Comment by Desertsnowman** [comment]
@aoloe I made it so that you can do almost what ever you need. if you want managing of entries, simple drop on the save to post type processor. You can then edit it as a normal post. I really need to document some of the cool stuff it does.

**Comment by aoloe** [comment]
i guess it's time to install it and see myself what it does :-)

**Comment by rainbat** [comment]
@Desertsnowman 
I would like to reuse the DB-Toolkit Configuration Arrays and work on the existing MySQL Tables - you think we could make kind of an importer for the config and stay on the existing MySQL Tables ?
Can't really go the Custom Post Type Way for everything ...

**Comment by Desertsnowman** [comment]
@rainbat whats you're reasoning to reuse the config arrays? Is it purely for compatibility?

**Comment by rainbat** [comment]
@Desertsnowman 
Yes one Application has about 30 Interfaces with a Total of about 200 Fields - and this in 3 Languages here in Switzerland, there is no way to rebuild all the Fields with copy-paste ... my finger would fall off half-way-through

**Comment by Desertsnowman** [comment]
Ok, good point

**Comment by Desertsnowman** [comment]
@rainbat - So been thinking about it over the last few weeks and think the best course of action is installing the current version on WP4.4 and then fixing it to be functional again and make an updated release. From there, I can start rebuilding the UI (because it sucks) and resurrect this plugin to be something useful again.
